src/marketing/price-rule-multiple-sku.md

```diff
@@ -2,53 +2,56 @@
 title: Price Rule with Multiple SKUs
 ---
 
-A single price rule can be applied to multiple SKUs, which makes it possible to create a variety of promotions based on a product, brand, or category. When creating this rule, you want to set conditions to match any products of selected SKUs. When building the rule, you can easily browse and select SKUs without needing to know all of your product SKUs from memory.
+A single catalog price rule can be applied to multiple SKUs, which makes it possible to create a variety of promotions based on a product, brand, or category. When creating this rule, you want to set conditions that match the selected SKUs. When building the rule, you can easily browse and select SKUs from the grid.
 
 ## Step 1. Verify Storefront Properties of Attribute
 
-Before you begin, make sure that the Storefront Properties of the SKU attribute are set to `Use in Promo Rules`.
+Before you begin, make sure that the Storefront Properties of the `sku` attribute are set to `Use in Promo Rules`.
 
 1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
 
-1. In the search filter at the top of the **Attribute** Column, enter `sku` and click <span class="btn">Search</span>.
+1. In the search filter at the top of the **Attribute Code** column, enter `sku` and click <span class="btn">Search</span>.
```

Comment:
i've been replacing bold with ital when describing a place on the screen, and using bold just for click items.

Reply:
oh yeah... thanks1